phoenix-ios/phoenix-ios/views/configuration/general/ElectrumConfigurationView.swift

```diff
@@ -106,8 +124,10 @@ struct ElectrumConfigurationView: MVIView {
 						Text("-")
 					}
 				}
-				
-				/* if let xpub = mvi.model.xpub {
+
+			/*	Why is this removed ?
```

Comment:
It was commented out because the way to access the xpub from the shared module changed. We now have a method `PhoenixBusiness.getXpub()`. Note that this method should be removed, and we should instead directly access the `walletManager` from the business, and get the xpub from the embedded  `Wallet` object.